### PR TITLE
UI auth pages

### PR DESCRIPTION
**Pull request by Brianna-Flo**
## Description

<what this pull request is doing>

- adding react component library to the auth page component which is used in the login and signup pages as well
- Using React component library in the dietary preference buttons presented on the signup page
- Using react component library for the account page since it reuses many of the modifications included in this PR
- Delete all CSS files from codebase

<what will be done in later PRs and not included here>

- Adding React UI components to the login page, registration page, and account page

## Milestones
<the milestones or stories/features that this works towards>

- migrating to react component library

## Resources
<links to tutorials, code snippets, inspirations>
<if you took or translated specific code from the internet, please call it out here!>

## Test Plan
<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>
https://docs.google.com/document/d/1u4QuX_0hLM_7-EKXGbdEtpxEErOixgLAIE5qFVQna7s/edit?usp=sharing

### CODE REVIEW

File: frontend/src/pages/LoginPage.tsx [hunk 1]
```diff
@@ -46,10 +46,13 @@ const LoginPage = () => {
   }
 
   return (
-    <div className="login-page">
+    // <Box className="login-page">
```
Comment (nithyad) [inline frontend/src/pages/LoginPage.tsx]: delete

File: frontend/src/pages/LoginPage.tsx [hunk 2]
```diff
@@ -46,10 +46,13 @@ const LoginPage = () => {
...
+    <Box>
       <AppHeader />
-      <section className="login-page">
-        <Box className="login-content">
+      {/* <Box className="login-page"> */}
```
Comment (nithyad) [inline frontend/src/pages/LoginPage.tsx]: also delete

File: frontend/src/pages/LoginPage.tsx [hunk 3]
```diff
@@ -46,10 +46,13 @@ const LoginPage = () => {
...
+      {/* <Box sx={{height: "100vh", display: "flex", justifyContent: "center", alignItems: "center"}}> */}
```
Comment (nithyad) [inline frontend/src/pages/LoginPage.tsx]: delete

**Review by nithyad — APPROVED** [review]
